docker/native_image/Dockerfile

```diff
@@ -87,5 +87,9 @@ VOLUME  ["/var/log/inmanta", "/var/lib/inmanta/"]
 # Change the workdir to the HOME directory of the inmanta user
 WORKDIR /var/lib/inmanta
 
+# Configure a healthcheck which validates that the api is responsive
+HEALTHCHECK --interval=10s --timeout=3s --start-period=30s --start-interval=5s --retries=3 \
+    CMD curl -fs http://${INMANTA_SERVER_BIND_ADDRESS}:${INMANTA_SERVER_BIND_PORT}/api/v1/serverstatus
```

Comment:
* We need to make sure this also works when https is enabled.
* I would increase the `--start-period` to 5 minutes, so that the status doesn't go to "unhealthy" when the database migration scripts take a long time to run.

Reply:
I believe everything should be addressed now